modules/badge/react/README.md

```diff
@@ -0,0 +1,91 @@
+# Canvas Kit React Badge
+
+CountBadge provides a quantity-based summary with dynamic values.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-react-badge
+```
+
+## Accessibility
+
+A common use case for the CountBadge is for displaying notifications, and there are several
+accessibility concerns you'll want to keep in mind:
+
+- The button should have an aria-label that updates with the count
+- The elements inside the button should have `aria-hidden`
+- The live region should be outside the button
+- The live region should be visually hidden and only contain text
+
+### Example
+
+```jsx
+<IconButton
+  style={{ position: "relative" }}
+  aria-label={`Alerts ${count} new notifications`}
+  size={IconButton.Size.Medium}
+  variant={IconButton.Variant.Circle}
+>
+  <SystemIcon icon={notificationsIcon} aria-hidden="true" />
+  <Badge count={count} aria-hidden="true" />
+</IconButton>
+<div className="wdc-accessible-hide" role="status" aria-live="polite" aria-atomic="true">
```

Comment:
Is this CSS class normally available? I don't know if we automatically include this class to be available for all React components

Reply:
That's true. In react-land, they would likely:

```tsx
import { accessibleHide } from "@workday/canvas-kit-react";

const Hidden = styled("div")({
  ...accessibleHide
});
```

and then use that component. I'll update the docs to show that. 👍

Reply:
updated 👍